src/webgpu/api/validation/image_copy/texture_related.spec.ts

```diff
@@ -89,7 +105,9 @@ g.test('usage')
   });
 
 g.test('sample_count')
-  .desc(`Multisampled textures cannot be copied.`)
+  .desc(
+    `Multisampled textures cannot be copied. Note that we don't test 2D array and 3D textures because multisample is not supported for 2D array and 3D texture creation`
```

Comment:
I assume this is changed in the other PR.

Reply:
Yeah. But that PR attempted to address another issue. In addition, as you pointed out, that PR might be unneeded because sample_count can be 4 only if it is T2T copy but this test is for writeTexture, B2T and T2B copy.